osquery/config/config.cpp

```diff
@@ -697,7 +772,8 @@ Status Config::update(const std::map<std::string, std::string>& config) {
     }
 
     if (!status.ok()) {
-      // The content was not parsed correctly.
+      LOG(ERROR) << "Content was not parsed correctly. " << source.first << ": "
```

Comment:
Need more context here. If I'm reading this from a log I have no idea what "Content" you're referring to.